### ISSUE TITLE
Tries to cleanup `/` when run under webpack-dev-server

### ISSUE THREAD

**Issue by jonathanperret** [gpbl/webpack-cleanup-plugin#7]
Warning: try this in a VM/container/throwaway machine. There is a risk that files will be removed from all over the disk!

With the example `webpack.config.js`:

``` js
{
  output: {
    path: "/my/output/path"
  },
  plugins: [
    // Remove all the files from /my/output/path that are not generated by the
    // the current instance of the webpack compiler
    new WebpackCleanupPlugin()
  ]
}
```

If you try running [webpack-dev-server](https://github.com/webpack/webpack-dev-server):

``` sh
$ npm install webpack-dev-server
$ ./node_modules/.bin/webpack-dev-server
```

You will end up with, if you're lucky, an error in `recursiveReaddirSync`:

```
Error: EBADF: bad file descriptor, lstat '/dev/fd/22'
    at Error (native)
    at Object.fs.lstatSync (fs.js:839:18)
    at …/node_modules/webpack-cleanup-plugin/node_modules/recursive-readdir-sync/index.js:12:16
    at Array.forEach (native)
    at recursiveReaddirSync (…/node_modules/webpack-cleanup-plugin/node_modules/recursive-readdir-sync/index.js:11:9)
    at …/node_modules/webpack-cleanup-plugin/node_modules/recursive-readdir-sync/index.js:14:26
    at Array.forEach (native)
    at recursiveReaddirSync (…/node_modules/webpack-cleanup-plugin/node_modules/recursive-readdir-sync/index.js:11:9)
    at /Users/Sebastien/code/boobsunit/boob_identity/node_modules/webpack-cleanup-plugin/node_modules/recursive-readdir-sync/index.js:14:26
    at Array.forEach (native) 
```

If you're unlucky, the scan will complete and `WebpackCleanupPlugin` will happily proceed with removing all the files from your disk.

This happens because `webpack-dev-server` forces the output path to `/`: https://github.com/webpack/webpack-dev-server/blob/5b94506/bin/webpack-dev-server.js#L142

Because `webpack-dev-server` uses an in-memory filesystem, it can get away with a `/` output path. Not so for `WebpackCleanupPlugin`, unfortunately.

**Comment by jonathanperret** [comment]
Note that at least one other plugin hit that issue: https://github.com/gajus/write-file-webpack-plugin/issues/1

**Comment by gpbl** [comment]
Thanks for reporting. I never tested this plugin with the dev server. 😰

I think the best solution is explicitly requiring  the path to delete from the constructor.

**Comment by kmcmichael-twist** [comment]
@gpbl Remove this repo and disable the npm package if you aren't going to fix this problem. You should have removed this in March until you addressed the issue.

**Comment by gpbl** [comment]
Yup @kmcmichael-twist sorry for the unforgivable delay 🙏
Released as v0.3.0 🎉
